### PR TITLE
docs: add nvs docs

### PR DESCRIPTION
**Pull request by MrJithil**
Added docs for `nvs`.

Issue: #3873 

Since the official certification [OpenJS Node.js Services Developer (JSNSD)](https://training.linuxfoundation.org/certification/jsnsd/) recommends using the `nvs` in windows machines but it lacks the in the `package manager` docs. 

After the changes (sample):
image

### CODE REVIEW

File: locale/en/download/package-manager.md [hunk 1]
```diff
@@ -237,6 +238,47 @@ from source:
 nvm uninstall 8
 ```
 
+## nvs
+While nvm is recommended for  Unix / OS X systems, and there is an unaffiliated `nvm-windows` version manager for Windows. The recommended version manager for Windows is `nvs`.
```
Comment (Trott) [inline locale/en/download/package-manager.md]: The other entries avoid endorsement. This recommendation seems inappropriate.

**Comment by Trott** [comment]
+1 to adding `nvs`. More just-the-facts and less endorsement/recommendation would be an improvement, though.

@nodejs/version-management

We should see about getting the `nvs` maintainer(s) in the `version-management` team if they're not already there.

File: locale/en/download/package-manager.md (hunk 1 above)
Comment (MrJithil) [reply locale/en/download/package-manager.md]: The below will be good?
"While nvm is recommended for  Unix / OS X systems, the recommended version manager for Windows is `nvs`."

Comment (MrJithil) [reply locale/en/download/package-manager.md]: I changed the sentence to **"`nvs` is a version manager for Windows."**

**Comment by jasonkarns** [comment]
> We should see about getting the `nvs` maintainer(s) in the `version-management` team if they're not already there.

@jasongin is in the team; unsure if there are other maintainers that should be invited.

File: locale/en/download/package-manager.md [hunk 2]
```diff
@@ -237,6 +238,47 @@ from source:
 nvm uninstall 8
 ```
 
+## nvs
+`nvs` is a version manager for Windows.
+
+The `nvs` version manager is actually cross-platform so can be used on  Unix / OS X systems as well but `nvm` is a lot more popular there.
```
Comment (ljharb) [inline locale/en/download/package-manager.md]: "actually" implies others aren't; `nvm` works on WSL, and also now cygwin, and git bash, and Powershell.

**Review by Trott — APPROVED** [review]
LGTM although here are a few other optional improvements:

* _OS X_ is an obsolete term. Use _macOS_ instead.
* Perhaps use _Unix-like_ instead of _Unix_.
* Change _release page here, https://..._ to just _release page_ and make it a link to the URL.

**Comment by shadowspawn** [comment]
I suggest do not mention `nvm` at all. You might choose to still focus on Windows in the description as less common to support Windows, but one of the features of nvs is being cross-platform.

e.g.

> The `nvs` version manager is cross-platform and can be used on Windows, macOS, and Unix-like systems

**Comment by ljharb** [comment]
@shadowspawn i totally agree; `nvs` should simply be described, without implying it's the best or only option for anyone/anything (just like all the other entries).

**Comment by MrJithil** [comment]
All the above comments have been added. Additionally, I have added a header4 for macOS/Unix-Like systems. 
The final out is here:
image

Or the URL: https://github.com/MrJithil/nodejs.org/blob/docsAddNvs/locale/en/download/package-manager.md#nvs

File: locale/en/download/package-manager.md (hunk 2 above)
Comment (MrJithil) [reply locale/en/download/package-manager.md]: Thanks for the feedback. I have added something similar to that.

**Comment by ghost** [comment]
@MrJithil：Fix a typo and then finish the translation for both zh-CN and zh-TW.

@nodejs/website ：Do not forget to do translations :)

**Review by ghost — APPROVED** [review]

**Comment by MrJithil** [comment]
> @MrJithil：Fix a typo and then finish the translation for both the-CN and zh-TW.
> 
> @nodejs/website ：Do not forget to do translations :)

We could have rebased the commits to a single one. Or at least squash it to keep it simple.

Unfortunately, it's still a typo. Let me fix it.  I have created a PR for that. #3881

**Comment by nschonni** [comment]
@MaledongGit I thought we had agreed before that translations don't belong in the same PR, because till the english lands, the text might change (and might mess up someone else' PR that they may still be working on)

**Comment by ghost** [comment]
@MrJithil & @nschonni ：I've merged it in the squach mode by GitHub in default, and it seems for the next time, I'll nofity you on the typo by creating another PR instead of fixing it directly here, thanks to tell me.

**Comment by nschonni** [comment]
@MaledongGit thanks for doing the translations 😄 ! sometimes it's just better to keep the activities separate

**Comment by max-hk** [comment]
How about [Volta](https://github.com/volta-cli/volta)? It seems functionally similar to nvs.

**Pull request merged**